theia-electron/scripts/theia-electron-main.js

```diff
@@ -0,0 +1,7 @@
+const path = require('path')
```

Comment:
@marechal-p 
I think we'll need to setup another plugins folder, using env var `THEIA_PLUGINS`, where the user can install additional ones, manually at first and through the new plugins widget soon. Any thoughts on that?

Reply:
Done, added it as:

```ts
// Lookup inside the user's home folder for more plugins
process.env.THEIA_PLUGINS = `local-dir:${path.resolve(os.homedir(), '.theia', 'plugins')}`
```

Reply:
For now uses will need to manually put the extra VS Code extensions they want in that folder. Mention in app's README?

Reply:
> For now users will need to manually put the extra VS Code extensions they want in that folder. Mention in app's README?

@marechal-p do you mind adding this small suggestion, then I think we can approve and merge 🍾

Reply:
Done.